sentences/en/media_player_HassMediaClearPlaylist.yaml

```diff
@@ -0,0 +1,8 @@
+language: en
+intents:
+  HassMediaClearPlaylist:
+    data:
+      - sentences:
+          - "(clear;<name>) (queue|playlist)"
```

Comment:
Would "delete" work as a verb here as well?
Also, I'd add `clear [the ](queue|playlist) <in> <name>`

Reply:
fixed